Makefile

```diff
@@ -34,13 +37,27 @@ run: data
 	$(PYTHON) -m user_guide.src.examples.window_functions
 	$(PYTHON) -m user_guide.src.examples.how_can_i.filter
 
+serve_user_guide: run
+	cd /usr/src/user_guide; mdbook serve --hostname 0.0.0.0 --port 8000
+
+serve_ref_guide_python: docs
+	cd /usr/src/reference_guide_python; mdbook serve --hostname 0.0.0.0 --port 8000
+
+serve: serve_user_guide
+
 clean:
 	-@rm -fr .venv
 	-@rm -fr data
 	-@rm -fr user_guide/src/outputs
 	-@rm -fr `find . -name __pycache__`
 	-@cd user_guide; mdbook clean
+	-@cd reference_guide_python; mdbook clean
+	-@rm -fr reference_guide_python/src
 
-fmt:
-	mdformat .
+# should really be off-loaded to pre-commit:
+# - isort is not ran
+# - running mdformat here is dangerous, some files *should* be excluded
+format:
 	black .
+	flake8 --max-line-length=88 --max-doc-length=88 .
+	mdformat .  # could add --wrap=88
```

Comment:
Read my comment above. Might end up doing something like `mdformat $(find . -name "*.md" | grep -v ...)` or something. `isort` is not fundamentally needed, but `black` doesn't take care of everything...